### ISSUE TITLE
[Question] discussion about multiple images?

### ISSUE THREAD

**Issue by jacekpoplawski** [haotian-liu/LLaVA#1072]
### Question

hello,
I just started to use llava 1.6 34B few days ago and it's fantastic, 

to me it's much better than ChatGPT, because ChatGPT refuses to load most photos,
I am a photographer and I am trying to discuss my photos with llm, to ask how to improve composition or postprocessing and llava is able to understand details of each photo, this is awesome

my question: is it possible to discuss about multiple images?
for example, I want to show unprocessed photo to the model and then show processed photo and ask what llm thinks, or show few photos and ask for best one

I was able to use llava 34b gguf by loading it into llama-cpp (llava-cli and server), I am not able to use unquantized model because I have only 24GB VRAM

**Comment by pseudotensor** [comment]
Yes, multiple images of order 10-20 like claude-3 (20), gpt-4-vision (10), or gemini-pro-vision (16) is crucial in order to do (e.g. ) document Q/A with images as various documents not able to be handled as just text.

**Comment by pseudotensor** [comment]
@haotian-liu Any plans, or can code be changed for fine-tuning?
